client/src/main/java/org/apache/rocketmq/client/impl/MQClientAPIImpl.java

```diff
@@ -3080,6 +3081,21 @@ public void setMessageRequestMode(final String brokerAddr, final String topic, f
         }
     }
 
+    public MessageRequestModeSerializeWrapper getAllMessageRequestMode(final String brokerAddr) throws MQBrokerException,
+            RemotingConnectException, RemotingSendRequestException, RemotingTimeoutException, InterruptedException {
+        RemotingCommand request = RemotingCommand.createRequestCommand(RequestCode.GET_ALL_MESSAGE_REQUEST_MODE, null);
+        RemotingCommand response = this.remotingClient.invokeSync(brokerAddr, request, 3000);
```

Comment:
may timeout when execute get all

Reply:
The local test does not time out, I can increase the timeout parameter if necessary